StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/LinkEnabledPaymentMethodElement.swift

```diff
@@ -35,38 +34,25 @@ final class LinkEnabledPaymentMethodElement: ContainerElement {
         return stackView
     }()
 
-    let paymentMethodType: STPPaymentMethodType
-
-    let paymentMethodElement: PaymentMethodElement
-
     let inlineSignupElement: LinkInlineSignupElement
 
     init(
-        type: STPPaymentMethodType,
-        paymentMethodElement: PaymentMethodElement,
         configuration: PaymentSheet.Configuration,
         linkAccount: PaymentSheetLinkAccount?,
         country: String?,
         showCheckbox: Bool
     ) {
-        self.paymentMethodType = type
-        self.paymentMethodElement = paymentMethodElement
         self.inlineSignupElement = LinkInlineSignupElement(
             configuration: configuration,
             linkAccount: linkAccount,
             country: country,
             showCheckbox: showCheckbox
         )
 
-        paymentMethodElement.delegate = self
         inlineSignupElement.delegate = self
     }
 
-    func makePaymentOption(intent: Intent, elementsSession: STPElementsSession) -> PaymentOption? {
-        guard let params = updateParams(params: .init(type: .stripe(paymentMethodType))) else {
-            return nil
-        }
-
+    func makePaymentOption(intentConfirmParams params: IntentConfirmParams) -> PaymentOption? {
```

Comment:
I think we can move this method implementation to `LinkInlineSignupElement`'s `func updateParams(params: IntentConfirmParams) -> IntentConfirmParams?`.  That's the normal way that Elements update the IntentConfirmParams, I don't know why we did it differently here.

Reply:
After that this class might not have a purpose anymore, it just contains `LinkInlineSignupElement`?